VideoLocker/res/values/errors.xml

```diff
@@ -2,47 +2,71 @@
 <resources>
 
     <!-- Network error messages -->
+    <!-- Error shown when  user attempts an action but needs an internet connection -->
     <string name="network_not_connected">You are not connected to the Internet.</string>
+    <!-- Concise error shown when user attempts an action but needs an internet connection -->
     <string name="network_not_connected_short">Network not reachable</string>
-    <string name="network_panel_dialog_message_help1">Please check your internet connection </string>
-    <string name="network_panel_dialog_help">This video can\'t load at the moment </string>
+    <!-- Concise error shown when user attempts an action but needs an internet connection -->
     <string name="network_panel_slow_dialog_message1">Please check your internet connection</string>
+    <!-- Alert dialog title shown when a network request fails -->
     <string name="reset_no_network_title">Connection Error</string>
+    <!-- Alert dialog content shown when a network request fails because the user has no internet connection -->
     <string name="reset_no_network_message">You are not connected to the Internet. Please check your Internet connection.</string>
-    <string name="network_panel_slow_dialog_message2">This page is taking longer than expected to load </string>
+    <!-- Title of prompt indicating issue loading a video -->
     <string name="network_error_header">Problem loading video</string>
+    <!-- Content of prompt indicating issue loading a video -->
     <string name="network_error_message">Please check your Internet Connection.</string>
```

Comment:
Probably a good idea. Sometimes it's hard to know with UI strings if two of them really want to refer to the same object or if you'll want them to translate differently in different contexts. I'm fine with us cleaning some up, but propose it be a separate PR.